src/main/java/pl/stormit/tinyurl/domain/model/UrlAnalytics.java

```diff
@@ -0,0 +1,37 @@
+package pl.stormit.tinyurl.domain.model;
+
+import lombok.*;
```

Comment:
I prefer explicit imports, especially when we have only few options.

Reply:
Corrected.